### PR TITLE
<inputWithOptions/> - fix onfocus triggered twice when suffix icon is clicked

### PR DESCRIPTION
**Pull request by roibWix**
### 🔦 Summary

...

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  Project Name: **wix-ui-tpa**

  You can find it under one of the following URLs:
  * https://bo.wix.com/pages/wix-ui-tpa.pr_6873/

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  Project Name: **wix-style-react**

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6873

**Review by EgoziE — CHANGES_REQUESTED** [review]
There's a Testkit function `clickMenuArrow` that I see we don't test along with spies onFocus/onBlur props.
Please write a test for it and compare before and after your change.

File: packages/wix-style-react/src/Input/test/Input.spec.js [hunk 1]
```diff
@@ -323,6 +323,17 @@ describe('Input', () => {
...
       });
+
+      it('should be called once when menu arrow is clicked', async () => {
+        const onFocus = jest.fn();
+        const onInputClicked = jest.fn();
+        const { driver } = render(
+          <Input menuArrow onFocus={onFocus} onInputClicked={onInputClicked} />,
+        );
+        await driver.clickMenuArrow();
+        expect(onFocus).toBeCalledTimes(1);
```
Comment (EgoziE) [inline packages/wix-style-react/src/Input/test/Input.spec.js:334]: Also add a "before" test:
```
expect(onFocus).not.toBeCalled();
expect(onInputClicked).not.toBeCalled();
```

**Review by EgoziE — CHANGES_REQUESTED** [review]
Only 1 comment, other than that I approve.

**Review by EgoziE — APPROVED** [review]

**Pull request merged**